src/commands/doctor.ts

```diff
@@ -53,18 +57,19 @@ export default class Doctor extends SfdxCommand {
   private filesWrittenMsgs: string[] = [];
 
   public async run(): Promise<SfDoctorDiagnosis> {
+    const { flags } = await this.parse(Doctor);
     this.doctor = SFDoctor.getInstance();
     const lifecycle = Lifecycle.getInstance();
 
-    const pluginFlag = this.flags.plugin as string;
-    const commandFlag = this.flags.command as string;
-    const outputdirFlag = this.flags.outputdir as string;
-    const createissueFlag = this.flags.createissue as boolean;
+    const pluginFlag = flags.plugin;
```

Comment:
this extra consts are unnecessary now that we have flag typing (no more `as string`)